public/image-picker.html

```diff
@@ -264,6 +264,18 @@
             </select>
           </div>
         </div>
+        <div class="gts hidden-fade">
+          <label for="isGts">How much stability would you like?</label>
+          <div class="select">
+            <select id="isGts" name="isGts">
+              <option disabled selected value>
+                Select your update stream
+              </option>
+              <option value="gts">Bluefin GTS (for New Users, based on Fedora 40)</option>
+              <option value="stable">Bluefin (or Enthusiasts, based on Fedora 41)</option>
```

Comment:
```suggestion
              <option value="stable">Bluefin (or Enthusiasts, based on Fedora 40)</option>
```

This will only stay 40 for a few more days so no big deal.